### DISCUSSION TITLE
Environment variables per user

### DISCUSSION THREAD

**Discussion by fcsonline** [coder/coder#9787] [Announcements]
Feature request:

It could be cool to have the capability to configure environment variables per user in Coder. This feature would empower users to customize their workspaces with some specific settings, configure secrets per user, etc. Those environment variables should be automatically injected in all new workspaces.

It would be great to be able to save and read them from the CLI.

What do you think?

**Comment by matifali** [comment]
There already exists a similar feature request. See https://github.com/coder/coder/issues/7087

**Comment by fcsonline** [comment]
Wow, exactly. It would be awesome to have this.
